docs/updating-postgres.md

```diff
@@ -0,0 +1,107 @@
+# Updating Postgres
+
+## Minor Versions
+
+When upgrading to a new minor version of Postgres, please follow these steps:
+
+`X` _is the major version of Postgres and_ `Y` _is the new minor version._
+
+1. Clone the Neon Postgres repository if you have not done so already.
+
+    ```shell
+    git clone git@github.com:neondatabase/postgres.git
+    ```
+
+1. Add the Postgres upstream remote.
+
+    ```shell
+    git remote add upstream https://git.postgresql.org/git/postgresql.git
+    ```
+
+1. Create a new branch based on the stable branch you are updating.
+
+    ```shell
+    git checkout -b my-branch REL_X_STABLE_neon
+    ```
+
+1. Tag the last commit on the stable branch you are updating.
+
+    ```shell
+    git tag "REL_X_$(expr $Y - 1)_neon"
```

Comment:
This might be easier to read, if you pick an actual minor version like 15.3 ->15.4 instead of X and Y at the top of the document, and use that example throughout the document.

Reply:
Sure, I will make that change.